lib/modules/apostrophe-soft-redirects/index.js

```diff
@@ -94,6 +99,15 @@ module.exports = {
       return url.replace(/^(https?:)?\/\/[^/]+/, '');
     };
 
+    self.buildQueryString = function(parameters) {
```

Comment:
This method will fail on nested data structures and doesn't escape the keys and values. Use `qs.stringify`